srcpkgs/hplip/template

```diff
@@ -27,7 +27,7 @@ make_dirs="/var/lib/hp 0755 root root"
 hostmakedepends="pkg-config automake libtool python3"
 makedepends="openssl-devel python3-devel libxml2-python3 cups-devel sane-devel
  ghostscript-devel net-snmp-devel libusb-devel libjpeg-turbo-devel dbus-devel
- avahi-libs-devel"
+ avahi-libs-devel zlib-devel"
```

Comment:
this is also broken without the autoconf update, drop it from here and create a new separate PR for this fix